exercises/practice/robot-simulator/robot_simulator.go

```diff
@@ -1 +1,42 @@
 package robot
+
+// See defs.go for other definitions
+
```

Comment:
robot-simulator has three different test files: (eg: [robot_simulator_step2_test.go](https://github.com/exercism/go/blob/main/exercises/practice/robot-simulator/robot_simulator_step2_test.go))

Should we add comments or something (eg: `// Step 1`) to signpost which parts of the stub file are linked to which test file?

Reply:
I think it is a good idea.

It's also probably worth to move the instructions that are in the test files to an `instructions.append.md` file, for better visibility.

Reply:
Good call! I'll move them across